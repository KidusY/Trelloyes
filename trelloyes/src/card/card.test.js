import React from 'react';
import ReactDOM from 'react-dom';
import Card from './card';
import renderer from 'react-test-renderer';

it('Card smoke Test ', () => {
	const div = document.createElement('div');
	ReactDOM.render(<Card />, div);
	ReactDOM.unmountComponentAtNode(div);
});

it('Card smoke snapshot test', () => {
	const card = renderer.create(<Card />).toJSON();
	expect(card).toMatchSnapshot();
});
