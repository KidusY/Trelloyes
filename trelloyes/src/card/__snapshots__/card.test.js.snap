// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card smoke snapshot test 1`] = `
<div
  className="Card"
>
  <div>
    <h3>
      Default Card
    </h3>
    <p>
      Default Content
    </p>
  </div>
  <button
    type="button"
  >
    delete
  </button>
</div>
`;
